tests/unit/ui/components/TestDropdownMenuItem.js

```diff
@@ -0,0 +1,71 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import DropdownMenuItem from 'ui/components/DropdownMenu/DropdownMenuItem';
+import Link from 'amo/components/Link';
+
+
+describe(__filename, () => {
+  it('renders a section when only `title` is specified', () => {
+    const item = shallow(<DropdownMenuItem title="A section" />);
+
+    expect(item).toHaveClassName('DropdownMenuItem');
+    expect(item).toHaveClassName('DropdownMenuItem-section');
+    expect(item).toIncludeText('A section');
+  });
+
+  it('cannot receive `to` and `onClick` props at the same time', () => {
+    expect(() => {
+      shallow(
+        <DropdownMenuItem
+          title="A section"
+          to="/"
+          onClick={jest.fn()}
```

Comment:
Interesting. I guess there's nothing wrong with this (it's just a dummy function) but it might be more consistent to use `sinon.stub()`, just because already use that everywhere and it's our usual dummy function, even when we aren't testing it being called.

Reply:
Typo 😢